modules/auxiliary/scanner/smb/smb_enum_gpp.rb

```diff
@@ -66,7 +66,9 @@ def check_path(ip, path)
       when 'STATUS_INSUFF_SERVER_RESOURCES'
         vprint_error("Host rejected with insufficient resources!")
       when 'STATUS_OBJECT_NAME_INVALID'
-        vprint_error("opening \\#{path} bad filename")
+        vprint_error("opening #{path.inspect} bad filename")
+      else
+        vprint_error("Server responded unexpected status code: #{e.status_code.name.inspect}")
```

Comment:
Adding this additional logging, as it was previously hiding the original `STATUS_INVALID_PARAMETER` information from the `verbose=true` logs

![image](https://user-images.githubusercontent.com/60357436/129919396-e1fa942a-77ad-414e-9a9d-bc5822561f45.png)